core/services/relay/evm/chain_reader.go

```diff
@@ -5,19 +5,25 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/ethereum/go-ethereum"
 	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
```

Comment:
This package is deprecated, we're trying to avoid using it in any new code, gradually changing everything over to a combination of the builtin "errors" and "fmt" packages.